### PR TITLE
Updated the moniker and made some minor changes.

### PR DESCRIPTION
**Pull request by isaiahwilliams**
Updated the moniker and made some minor changes.

### CODE REVIEW

**Comment by EdisonDa** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/partner-center-java-docs/blob/dev-minor-updates/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-.openpublishing.publish.config.json)
[]() | :warning:Warning | | [Details](#user-content-)

<a name=".openpublishing.publish.config.json"></a>
### [.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/partner-center-java-docs/blob/dev-minor-updates/.openpublishing.publish.config.json)
 - **[Error]** Moniker partnercenter-1.8.1 mentioned in moniker range partnercenter-1.8.1 doesn&#39;t exist in moniker service. Depot partner-center-java-docs content won&#39;t be published
Learn more: https://review.docs.microsoft.com/en-us/new-hope/specs/ops/conceptual-versioning?branch=master

<a name=""></a>
### []()
 - **[Warning]** Your service account opbld18 does not have &#39;Write&#39; permission to this repo, so we used the fallback account EdisonDa. To ensure your PRs will use the service account in the future, please grant that account &#39;Write&#39; permission to this repo.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C22%5C32dbdb78-ef3e-d720-6668-27874f7b2281%5CPullRequest%5C201810221858256430-3%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eW5ENU%2B8RQ41dt5Gdz6QYzkiemzti5wxdZd8pMUelj0%3D&st=2018-10-22T18%3A53%3A45Z&se=2018-11-22T18%3A58%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/toc.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - name: Partner Center Java SDK
   href: overview.md
   items:
-    - name: Test MD
+    - name: Test
```
Comment (EdisonDa) [inline docs-ref-conceptual/toc.yml]: Do you want to rename this file, I name it "Test MD" as a placeholder there. You can access https://review.docs.microsoft.com/en-us/java/partnercenter/overview?view=partnercenter-1.8.1&branch=dev-minor-updates to check how does it like now.

File: partner-center-breadcrumb/toc.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-- name: Docs
+- name: PartnerCenter
```
Comment (EdisonDa) [inline partner-center-breadcrumb/toc.yml]: If we rename it to some others, we should provide an herf. Now it goes to 404 page. You can check https://review.docs.microsoft.com/en-us/java/partnercenter/overview?view=partnercenter-1.8.1&branch=dev-minor-updates

**Comment by isaiahwilliams** [comment]
@EdisonDa that was a good catch on the TOC. I reverted that change and replace the test page with get started page. Let me know if you see anything else that should be changed.

**Pull request merged**